src/AddItem.js

```diff
@@ -3,28 +3,73 @@ import firebase from 'firebase';
 
 const AddItem = () => {
   const [inputValue, setInputValue] = useState();
+  const [frequency, setFrequency] = useState(7);
+  const [success, setSuccess] = useState(false);
 
   const addItem = event => {
     event.preventDefault();
+
     return firebase
       .firestore()
       .collection('items')
-      .add({ name: inputValue });
+      .add({
+        token: '143',
+        name: inputValue,
+        frequency: frequency,
+        lastPurchased: null,
+      })
+      .then(setSuccess(true));
+  };
+
+  const handleOnChange = e => {
+    setFrequency(parseInt(e.target.value));
```

Comment:
I might suggest looking into adding the `radix` parameter of 10 here to your `parseInt` to make sure we're always getting the number that we'd expect. You can learn more about that over here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt

Reply:
Perfect! I added it in as well as updated the success message to include the item like Scott suggested.